File: joshua.cpp
#include <iostream>
# include<string>
using namespace std;

class myneighbour {
public:
    string name;
    string colour;
    int age;
    string tribe;
    string course;
};
int main() {
    myneighbour neighbour;

    neighbour.name="dennis kimtai";
    neighbour.colour="black";
    neighbour.age=21;
    neighbour.tribe="luhya";
    neighbour.course="mechanical engineering";
    
    cout << neighbour.name << " is " << neighbour.colour << " in colour" << endl;
    cout <<  neighbour.name << " is " << neighbour.age << " years old" <<endl;
    cout << neighbour.name << " is " << neighbour.tribe << endl;
    cout << neighbour.name << " is " << neighbour.course << endl;

    return 0;
}

File: joshua.py
class MyNeighbour:
    def _init_(self):
        self.name = ""
        self.colour = ""
        self.age = 0
        self.tribe = ""
        self.course = ""

# Create an object
neighbour = MyNeighbour()

neighbour.name = "dennis kimtai"
neighbour.colour = "black"
neighbour.age = 21
neighbour.tribe = "luhya"
neighbour.course = "mechanical engineering"

print(f"{neighbour.name} is {neighbour.colour} in colour")
print(f"{neighbour.name} is {neighbour.age} years old")
print(f"{neighbour.name} is {neighbour.tribe}")
print(f"{neighbour.name} is studying {neighbour.course}")